model_one/cli.py

```diff
@@ -1,198 +1,274 @@
-from array import array
+import os
 import json
-from re import M
-from time import time
 import requests
-import os.path
-import urllib.parse
 
-import os
+from enum import Enum
+from functools import wraps
+from typing import Optional, List, Union
+
+from pandas import Series
+
 
 API_KEY = os.environ.get("BEYONDML_API_KEY")
-api_url = 'https://api.beyond.ml'
+MINIMUM_ENTRIES = 32
 
-def _models_api():
-    return '{}/v0/models'.format(api_url)
 
+class BeyondmlModelException(RuntimeError):
```

Comment:
It is a good thing to create your own types of exceptions
We could have gone further and created separate exception classes for the majority of entities in this module, but I did not feel that it is necessary while they are in a single file